src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -3,57 +3,55 @@ import React, { useState, useEffect } from 'react';
 import AddIncident from './AddIncident';
 import DashboardTop from './DashboardTop';
 import Incidents from './Incidents';
-import ApprovedIncidents from './ApprovedIncidents';
 import './AdminDashboard.css';
 import { Modal } from './Modal';
 import useOktaAxios from '../../hooks/useOktaAxios';
 
 import {
   getData,
-  sortApproved,
   putIncidents,
   getLatAndLong,
+  getApprovedIncidents,
+  getPendingIncidents,
+  getFormResponses,
+  splitIncidentsByIds
 } from '../../utils/DashboardHelperFunctions.js';
 
-import axios from 'axios';
-import { useHistory } from 'react-router-dom';
+import IncidentStatus from './IncidentStatus';
 
 const AdminDashboard = () => {
-  // setting up local state to keep track of selected/"checked" incidents
-  const [selected, setSelected] = useState([]);
+  /** List of selected (checked) incident_ids */
+  const [selectedIds, setSelectedIds] = useState([]);
   const [allSelected, setAllSelected] = useState(false);
-  const [incidents, setIncidents] = useState([]);
 
-  //   setting state necessary for pagination
-  const [pageNumber, setPageNumber] = useState(1);
-  const [incidentsPerPage, setIncidentsPerPage] = useState(10);
-  const [currentSet, setCurrentSet] = useState([]);
-
-  //   setting state for confirmation buttons of confirming/rejecting
-  const [confirmApprove, setConfirmApprove] = useState(false);
```

Comment:
I know a lot of this state was being used, so it's possible in your reformatted version this component state is no longer necessary, but I'd love to see all this functionality restored through your reformat in a loom video walkthrough prior to approving for merge.

Reply:
Yeah, I realize there's a lot of stuff cut out in here. I'm about to work on a video now.